ansible_tower_client.gemspec

```diff
@@ -17,15 +15,14 @@ Gem::Specification.new do |spec|
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   spec.bindir        = "exe"
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
-  spec.require_paths = ["lib"]
```

Comment:
hmm, why remove this?

Reply:
It's already the default: https://guides.rubygems.org/specification-reference/#require_paths=

Reply:
🤷‍♂️ why do we do these things?  Lol

Reply:
maybe it wasn't the default at one time or we had more than just lib...

Reply:
I see this line copied around a lot in gemspecs out of habit I think.

Reply:
> why do we do these things? Lol

We did this because gem builders do this...same with the other changes in this PR